packages/rainbowkit/src/components/ProfileDetails/ProfileDetails.tsx

```diff
@@ -59,6 +61,7 @@ export function ProfileDetails({
   }
 
   const accountName = ensName ? formatENS(ensName) : formatAddress(address);
+  // @ts-ignore
   const ethBalance = balanceData?.formatted;
```

Comment:
TODO: Wagmi has deprecated the `formatted` value. I'll have to see how we can format balances.